src/components/GetImageForm.js

```diff
@@ -69,7 +67,7 @@ export default class GetImageForm extends Component{
       </select>
       <label htmlFor="sol">Martian Sol: 1000-2000</label>
       <input type="number" onChange={this.handleSol} max="2000" min="1000" value={this.state.value}/>
-      <GetImageButton {...this.state} />
+      <GetImageButton {...this.state} onClick={this.fetchRoverImage} />
```

Comment:
Finally, when the button is clicked, we want to fetch rover images. 

In real life, this is superfluous. The GetImageButton only has a single input, so wrapping it in a component is more confusing than what you gain.

Now that I look at it, nothing is being done with those props, so the `{...this.state}` bit is redundant as well. The only prop this cares about is `onClick`, so that part could be removed.